packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/FundConfirmationView.svelte

```diff
@@ -5,10 +5,16 @@
     import { onMount, onDestroy } from 'svelte'
     import { formatTokenAmountBestMatch, selectedWallet, selectedWalletAssets } from '@core/wallet'
     import { activeProfile } from '@core/profile'
+    import { Balance } from '@iota/sdk/out/types'
 
     // TODO: use this output to calculate mana
     export let outputId: string | undefined
 
+    let walletBalance: Balance | undefined
+
+    $: formattedWalletBalance = walletBalance
+        ? formatTokenAmountBestMatch(Number(walletBalance.baseCoin?.available), baseCoin?.metadata)
```

Comment:
I have my doubts about this, the output amount is being shown instead of the wallet balance.
I think we have to show `the available balance of the wallet `- `the sum of the rest of the implicit accounts`